lib/rubocop/cop/chef/deprecation/log_resource_notifications.rb

```diff
@@ -19,7 +19,7 @@ module RuboCop
   module Cop
     module Chef
       module Deprecations
-        # In Chef Infra Client 16 the log resource no longer notifies when logging so notifications should not be triggered from log resources. Use the notify_group resource introduced in Chef Infra Client 15.8 instead to aggregate notifications.
+        # In Chef Infra Client 16 and later the log resource no longer notifies when logging so notifications should not be triggered from log resources. Use the notify_group resource introduced in Chef Infra Client 15.8 instead to aggregate notifications.
```

Comment:
Notifications triggered by the log resource are deprecated beginning with Chef Infra Client 16 (April 2020). The log resource no longer notifies when logging and cannot be used successfully to trigger notifications from the log resource. Use the notify_group resource introduced in Chef Infra Client 15.8 for notifications.

I don't know how important `aggregate` is for describing this.

Reply:
it's not deprecated in 16. it's just gone